ci/replace_string.py

```diff
@@ -0,0 +1,51 @@
+#!/usr/bin/python3
+
+import argparse
+
+
+def replace_string_in_file(file_path: str, original_str: str, new_str: str):
+    # Read the file
+    with open(file_path, "rb+") as file:
+        content = file.read()
+
+        # Check if the original string exists in the file
+        if original_str.encode() not in content:
```

Comment:
Maybe we should also check that the string in question in present only in one place?